src/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -179,7 +179,6 @@ public String remoteShutdown(final String hashedPassword, final String salt) {
     if (encryptedPassword.equals(hashedPassword)) {
       (new Thread(() -> {
         System.out.println("Remote Shutdown Initiated.");
-        ThreadUtil.sleep(1000);
```

Comment:
@ron-murhammer I'm not sure if it is actually safe to remove the thread. I welcome that as a follow on task. It's quite clear we do not need the sleep.